docs/codehealth_rotation.md

```diff
@@ -0,0 +1,87 @@
+# Code Health
+
+## Overview
+
+This is an outline for the CodeHealth rotation, which aims to proactively manage
+technical debt and maintain our codebase by keeping it up-to-date with upstream
+Chromium. These tasks are essential for keeping the codebase current, clean, and
+maintainable.
+
+Additionally, CodeHealth work is a great opportunity to familiarise oneself with
+different areas of the codebase, modernising the code, introducing new language
+features, and applying the style guide's best practices.
+
+## Rotation Process
+
+To ensure steady, incremental improvements, a weekly rotation is in place.
+
+  * **Staff**: Each `brave-core` pod provides a different volunteer to take up
```

Comment:
Some folks on my team don't do `brave-core` work. I'm guessing there's flexibility in who I can assign for this?

Reply:
I've tweaked the wording a bit here to to clarify that only `brave-core` engineers are supposed to participate.

Reply:
sgtm and flexibility of when in each calendar month it is performed.

Reply:
I've updated the documentation to reflect up to 8hrs and minimum of a PR.